learning-content/beginners-guides/evalopensourcevspropietary.md

```diff
@@ -0,0 +1,149 @@
+# Evaluating Open Source versus Proprietary Software for Data Workflows in the Social Sector: A Beginner’s Guide
+### Stephanie Coker and Benjamin Kinsella
+
+##### Introduction:
+Given the central role digital technologies play across many industries, the social sector has benefited enormously from Open Source software (OSS). In 2020, GitHub, the world’s largest internet hosting for software development, observed over 1.9 billion contributors to projects, representing a 25% increase in open source repositories from 2019. The proliferation of OSS offers social sector leaders and practitioners the opportunity to innovate and collaborate using digital technologies, providing novel alternatives to proprietary software.
```

Comment:
Can you rephrase this opening paragraph using the language on the github.com homepage - "the largest software development platform in the world" - or similar could work.  Also not that it's 1.9 **contributions,** not individual contributors.

Reply:
Done. Thanks!